packages/drizzle/src/createVersion.ts

```diff
@@ -29,14 +29,20 @@ export async function createVersion<T extends TypeWithID>(
     delete version.id
   }
 
+  const data: Record<string, unknown> = {
+    autosave,
+    latest: true,
+    parent,
+    version,
+  }
+
+  if (collection.timestamps && 'createdAt' in version) {
```

Comment:
Timestamps are required to be on for versioned collections. Unless this is just to make typescript happy, L40 doesn't need to be in this condition since that will always be available.

Reply:
`collection.timestamps` isn't required here, yes
but still needs to be checked for `createdAt`
![image](https://github.com/user-attachments/assets/7a24188f-0d82-4a1d-910c-0ac2a8f34660)